### ISSUE TITLE
508 Defect Level 3, Screen Readers, PDF document does not have headings

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#24604]
Label: 24145 

    - [508-defect-3]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3

Issue Title: 508 Defect Level 3, Screen Readers, PDF document does not have headings
 Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a screen reader user, I expect that the logical headers within a PDF document will be marked up as headings. 
<h2>Details</h2>
When I complete the pre-appointment questionnaire and download the PDF containing my answers, there are no headings which would allow me to easily skip from section to section. 
<h2>Acceptance Criteria</h2>
The title of the document is tagged with a level 1 heading.
Veteran Information and Prepare for Your Appointment are tagged with level 2 headings.
